classes/UpgradeContainer.php

```diff
@@ -84,16 +84,22 @@
  */
 class UpgradeContainer
 {
-    const WORKSPACE_PATH = 'workspace'; // AdminSelfUpgrade::$autoupgradePath
+    const WORKSPACE_PATH = 'workspace';
     const BACKUP_PATH = 'backup';
     const DOWNLOAD_PATH = 'download';
-    const LATEST_PATH = 'latest'; // AdminSelfUpgrade::$latestRootDir
-    const LATEST_DIR = 'latest/';
     const LOGS_PATH = 'logs';
+
+    // temporary folders
     const TMP_PATH = 'tmp';
+    const FILES_PATH = 'files';
+    const FILES_DIR = 'files/';
```

Comment:
Shouldn't these constants be called 

```suggestion
    const TMP_FILES_PATH = 'files';
    const TMP_FILES_DIR = 'files/';
```

like the 3 next ones? As they are all subdirectories of the `tmp/` folder.